R/CageData_Integration.R

```diff
@@ -0,0 +1,208 @@
+#' Converts different type of files to Granges
+#' Only Accepts bed files for now, standard format from Fantom5
+#' @param x An imported bed-file, to convert to GRanges
+#' @param bed6 If bed6, no meta column is added
+bedToGR <- function(x, bed6 = TRUE){
+
+  if (!bed6){
+    gr <- GRanges(x[, 1],IRanges(x[, 2]-1, x[, 3]))
```

Comment:
for variables after comma put space eg. GRanges(x[, 1], IRanges(x[, 2]-1, x[, 3]))